gerrit/changes/change.py

```diff
@@ -0,0 +1,124 @@
+"""
+Change
+======
+
+Manage gerrit changes
+"""
+
+from gerrit.helper import decode_json
+from gerrit.error import UnhandledError
+from gerrit.projects.project import Project
+
+def add_change(gerrit_con, project, subject, branch, options):
```

Comment:
tbh i feel that this add_change should be part of the Change class

example work flow:
```
change = Change()
change.create(...)
change.add_reviewer(...)
```